workflows/iuf/operations/extract-release-distributions.yaml

```
@@ -49,8 +49,15 @@ spec:
           productName="{{=jsonpath(inputs.parameters.global_params, '$.product_manifest.current_product.name')}}"
           echo "Extract from: $tarfile for product: $productName"
           dir=$(echo "$tarfile" | sed 's|\(.*\)/.*|\1|')
```

Why not use `dirname`?

```suggestion
          dir=$(dirname "$tarfile")
```